Packs/Threatgraph/Integrations/Threatgraph/Threatgraph.yml

```diff
@@ -0,0 +1,129 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Threatgraph
+  version: -1
+configuration:
+- defaultvalue: https://falconapi.us-2.crowdstrike.com
+  display: Server URL (e.g. https://falconapi.us-2.crowdstrike.com)
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
```

Comment:
This integration does not seem to support fetch-incident. This may confuse clients, please remove.